drivers/lutron/quantum.cr

```diff
@@ -0,0 +1,54 @@
+require "placeos-driver"
+require "quantum"
+
+class Lutron::Quantum < PlaceOS::Driver
+  descriptive_name "Lutron Quantum Gateway"
+  generic_name :Lighting
+  uri_base "https://engineeringwebdemo01.lutron.com/"
+
+  alias Client = ::Quantum::Client
+
+  default_settings({
+    api_key:    "ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+    device_key: "ab22c585-14e6-4c6b-b418-166728bcc608",
+  })
+
+  @client : Client = Client.new(host_name: "", api_key: "", device_key: "")
```

Comment:
This will raise if accessed before `on_update` is called, but that shouldn't happen. I think it's a bit neater this way.
```suggestion
  protected getter! client : Client
```

Reply:
Done.